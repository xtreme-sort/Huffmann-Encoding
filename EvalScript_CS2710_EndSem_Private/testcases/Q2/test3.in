DeriveHuffmanTree
3
5 01
8 1
T 00
Quit
