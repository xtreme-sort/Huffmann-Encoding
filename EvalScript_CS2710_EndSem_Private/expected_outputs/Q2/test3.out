((T,5),8);
